src/ssh_cert_service/views/api_v1/api-spec/paths/key/sign.yml

```diff
@@ -0,0 +1,30 @@
+post:
+  summary: Sign new CA  
```

Comment:
Should be 'Sign public key' .... This currently reads like we are signing a new certificate Authority (CA)